src/app/components/Disclaimer/index.test.tsx

```diff
@@ -43,60 +39,51 @@ const renderComponent = (
         },
       }}
     >
+      {/* @ts-expect-error - to diagnose */}
```

Comment:
I think this error is happening because the ServiceContext.Provider isn't expecting the value prop. The tests run and pass. I couldn't work out how to refactor this to use Types and also get the tests to pass.

Reply:
Are you able to use ServiceContextProvider with props instead?

Reply:
No I couldn't get that to work either. If I change  ServiceContext.Provider to  ServiceContextProvider then all the tests fail. I think this is because ServiceContextProvider is only expecting props: `children`, `service`, `variant`, `pageLang`. 

I don't know how to refactor this to pass in the Disclaimer Fixture data, since that is taken from the Config files and not passed in to the component as content.